memcached-operator/pkg/apis/cache/v1alpha1/zz_generated.defaults.go

```diff
@@ -0,0 +1,16 @@
+// +build !ignore_autogenerated
```

Comment:
@AlexNPavel @hasbro17 not sure if we want to add this file since we're [removing `defaulter-gen`](https://github.com/operator-framework/operator-sdk/pull/1288).

Reply:
Yeah we can remove this file since we won't be generating it soon.